dev-qt/qtwebengine/qtwebengine-5.14.1.ebuild

```diff
@@ -78,6 +78,8 @@ DEPEND="${RDEPEND}
 	pax_kernel? ( sys-apps/elfix )
 "
 
+PATCHES=( "${FILESDIR}/${P}-disable-fatal-warnings.patch" )
```

Comment:
can you put the bug number in a comment here please?

Reply:
sure, done